docs/source/noise/column_noise.rst

```diff
@@ -4,18 +4,128 @@
+lists of options that correspond to each column.
+All wrong options are equally likely.
+
+This noise type is called :code:`choose_wrong_option` in the configuration.
+It takes one parameter:
+
+.. list-table:: Parameters to the :code:`choose_wrong_option` noise type
+  :widths: 1 5 1
+  :header-rows: 1
+
+  * - Parameter
+    - Description
+    - Default
+  * - :code:`cell_probability`
+    - The probability that, for a cell in the column being configured, the wrong option is chosen.
+      The probability of this noise type is the same for all cells in a column;
+      it is not correlated with any attributes.
+    - 0.01 (1%)
+
+Write the wrong ZIP code digits
+-------------------------------
+
+When reporting a ZIP code on a survey or form, people may misremember or misreport
+the digits.
+They are probably more likely to get confused about the last few digits (which identify
+the small, specific area) than the first few (which will be the same over a larger area).
+
+This noise type is called :code:`write_wrong_zipcode_digits` in the configuration.
+It takes two parameters:
+
+.. list-table:: Parameters to the :code:`write_wrong_zipcode_digits` noise type
+  :widths: 1 5 1
+  :header-rows: 1
+
+  * - Parameter
+    - Description
+    - Default
+  * - :code:`cell_probability`
+    - The probability of a cell being *considered* to have this noise type.
+      One way to think about this is the probability that a ZIP code is reported by someone who isn't sure of their ZIP code.
+      This probability is the same for all cells in a column; it is not correlated with any attributes.
+      Whether or not there are actually any errors depends on the next parameter.
+    - 0.01 (1%)
+  * - :code:`digit_probabilities`
+    - A list of probabilities, one for each digit in a (5-digit) ZIP code.
+      The first item in this list is the probability **in a cell considered for this noise type** that the first digit of the ZIP
+      code will be wrong, the second item is the same probability for the second digit, and so on.
+    - [0.2, 0.2, 0.2, 0.36, 0.36]
 
 Misreport age
 -------------
 
+When someone reports their age in years, or especially when someone reports the age of someone else such as a family member,
+they may not get the value exactly right.
+When this noise type occurs, the reported age is off by some amount, for example a year or two older or younger than the
+person actually is.
+
+This noise type is called :code:`misreport_age` in the configuration.
+It takes two parameters:
+
+.. list-table:: Parameters to the :code:`misreport_age` noise type
+  :widths: 1 5 1
+  :header-rows: 1
+
+  * - Parameter
+    - Description
+    - Default
+  * - :code:`cell_probability`
+    - The probability that, for an age value in the column being configured, the age is misreported.
+      The probability of this noise type is the same for all cells in a column;
+      it is not correlated with any attributes.
```

Comment:
It's getting pretty redundant to say this in every parameter table. Since all the noise types (except Borrowed SSN I guess) have a cell probability, and it's never correlated with anything, can we just say this once at the top of the page or something?